Order history, Templating, Payment and Price engine section update (#9)
ezsystems/commerce-documentation: docs/guide/payment/payment_faq.md

The compiler pass is defined in the checkout bundle: `\Siso\Bundle\CheckoutBundle\DependencyInjection\Compiler\PaymentMethodsPass`

## How can I customize the payment options?

> To customize payment options displayed on the box payment options

Reviewer: ```suggestion
To customize payment options displayed in the payment options box
```

(e.g. to hide some if products are not available), change the payment form.

To change the shipping/payment form, replace or rewrite the form type service.

> The form type for the options choice is implemented in the checkout bundle in `\Siso\Bundle\CheckoutBundle\Form\Type\CheckoutShippingPaymentType`

Reviewer: ```suggestion
The form type for the options choice is implemented in the checkout bundle in `\Siso\Bundle\CheckoutBundle\Form\Type\CheckoutShippingPaymentType`.
```

The methods `CheckoutShippingPaymentType::buildForm()` and `CheckoutShippingPaymentType::setDefaultOptions()` are called to assemble the HTML form.

See [Configuration for the Checkout Forms](../checkout/checkout_api/forms/configuration_for_checkout_forms.md)

> and [Symfony's form documention](http://symfony.com/doc/current/book/forms.html) for more information.

Reviewer: ```suggestion
and [Symfony's form documention](http://symfony.com/doc/3.4/book/forms.html) for more information.
```

> ## Can I extend the size limit on orders?

Reviewer: ```suggestion
## Can I extend the size limit of orders?
```

> ## Can I extend the size limit on orders?

Author: ```suggestion
## Can I extend the order size limit?
```
